bin/inference/pycbc_inference

```diff
@@ -70,6 +81,13 @@ parser.add_argument("--samples-file", default=None,
                          "The priors must allow encompass the initial "
                          "positions from the InferenceFile being read.")
 
+# calibration data options
+parser.add_argument("--calib", action="store_true", default=False,
```

Comment:
Instead of putting this on the command line, I think this could go nicely in the configuration file.

For example, make a section
```
[calibration]
name = physical_model
transter-function-a-tst = /path/to/transfer/function
etc.
```
and if there is no ``[calibration]`` section, then don't use it. Have a ``name`` section that maps to the ``Recalibrate`` class you created. For example, in ``pycbc.distributions.__init__`` we have a ``dict`` that maps between the classes and names in the config file.